tests/integration/standard/test_connection.py

```diff
@@ -172,8 +172,8 @@ def fetch_connections(self, host, cluster):
                     if conn._connections is not None and len(conn._connections) > 0:
                         connections.append(conn._connections)
                 else:
-                    if conn._connection is not None:
-                        connections.append(conn._connection)
+                    if conn._connections and len(conn._connections.values()) > 0:
+                        connections.append(conn._connections.values())
```

Comment:
Is the intention for `conn` to be a list of lists?

Reply:
Yeah you are right, it was unintentional. Should use extend here

Reply:
for my deference, I was copying both from 2 line above :)

now fixing it